images/app-cpp/Dockerfile

```diff
@@ -39,6 +39,9 @@ RUN rm -fr /stage
 
 ENV RUN_SDK_AGENT=1
 
+RUN mkdir -p /opt/antaris/inbound
+RUN mkdir -p /opt/antaris/outbound
```

Comment:
@rahulbhivare  I would actually expect the user to create these directories implicitly through a docker (bind mount) volume. What is the use case to create these within the image itself? I don't see how one would access these directories from the host if not using a volume.

Reply:
Ok. So, I will remove this from here and add in README.md file